src/main/java/com/zaxxer/hikari/pool/HikariPool.java

```diff
@@ -573,7 +573,7 @@ public Boolean call() throws Exception
 
             // failed to get connection from db, sleep and retry
             quietlySleep(sleepBackoff);
-            sleepBackoff = Math.min(connectionTimeout / 2, (long) (sleepBackoff * 1.3));
+            sleepBackoff = Math.min(connectionTimeout / 5, (long) (sleepBackoff * 1.3));
```

Comment:
If I were writing a high-traffic website, like Wix, I would absolutely set ``connectionTimeout`` to 1000ms.  Front-end users experiencing >1s delays tend to walk away.  According to [KISSMetrics](https://blog.kissmetrics.com/loading-time/), *"40% of people abandon a website that takes more than 3 seconds to load."*  A 1000ms connection timeout allows the application layer to failover or switch pools before abondonment occurs.

That said, if database connections are encrypted (SSL) then 200ms is the typical minimum setup time for a connection, and the one core will be at nearly 100% for the duration of the setup.  A backoff that never goes above 200ms will choke one core completely.